src/Microsoft.Azure.WebJobs.Host/Scale/TargetScalerStatusContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Context used by <see cref="ITargetScaler.GetScaleResultAsync(TargetScalerStatusContext)"/> to decide
+    /// scale result.
+    /// </summary>
+    public class TargetScalerStatusContext
+    {
+        /// <summary>
+        /// The current worker dyanimc worker concurrency. 
```

Comment:
"The per instance concurrency for the associated function"

In remarks we can say "When not specified, the scaler will determine the concurrency based on configuration."